WordPress/WordPressUITests/Screens/Editor/FeaturedImageScreen.swift

```diff
@@ -0,0 +1,20 @@
+import Foundation
+import XCTest
+
+class FeaturedImageScreen: BaseScreen {
+
+
+    let navigationBar: XCUIElement
+
+    init() {
+        let app = XCUIApplication()
+        navigationBar = app.navigationBars["Featured Image"]
```

Comment:
Since the navigation bar titles are localized strings, it would be nice to use something else with an accessibility identifier to verify we're on the correct screen. (We'd like to be able to run UI tests in any language.)

I'd suggest using the "Remove Featured Image" button instead, since it's specific to this screen and you know you'll need to use it on this screen.